3-ecommerce-db/projects/cyf-ecommerce-api/server/package.json

```diff
@@ -0,0 +1,17 @@
+{

```

Comment:
no need to use yarn with this project. 
we don't deal with monorepo here 
where client and server run in parallel